pages/linux/extrepo.md

```diff
@@ -0,0 +1,21 @@
+# extrepo
+
+> Manage external Debian repositories.
+> It is used to manage external repositories in Debian.
+> More information: <https://manpages.ubuntu.com/manpages/focal/man1/extrepo.1p.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/extrepo.1p>.
```
We use manned as a default fallback when the author's documentation isn't present. (Btw if you referring to manpages.ubuntu.com in future, I would suggest removing the release from URL so that it automatically redirects to the latest one i.e  https://manpages.ubuntu.com/manpages/man1/extrepo.1p.html)